TileStache/Core.py

```diff
@@ -144,12 +144,21 @@
 """
 
 import logging
+from sys import modules
 from wsgiref.headers import Headers
-from StringIO import StringIO
-from urlparse import urljoin
+try:
+    from io import BytesIO
```

Comment:
Generally, TileStache deals with bytes rather than strings, since it's typically moving image or binary data around. I modified every instance of StringIO revealed by the tests, so I'm guessing the one in VecTiles/client.py is not covered.